1.2.0-alpha.2/CarPlay.html

```diff
@@ -0,0 +1,1043 @@
+                    <section class="section">
+                      <div class="abstract">
+                        <p><code>CarPlayManager</code> is the main object responsible for orchestrating interactions with a Mapbox map on CarPlay.</p>
+
+<p>Messages declared in the <code>CPApplicationDelegate</code> protocol should be sent to this object in the containing application&rsquo;s application delegate. Implement <code><a href="Protocols/CarPlayManagerDelegate.html">CarPlayManagerDelegate</a></code> in the containing application and assign an instance to the <code>delegate</code> property of your <code>CarPlayManager</code> instance.</p>
+<div class="aside aside-note">
+    <p class="aside-title">Note</p>
+    It is very important you have a single <code>CarPlayManager</code> instance at any given time. This should be managed by your <code>UIApplicationDelegate</code> class if you choose to supply your <code>accessToken</code> to the <code><a href="Classes/CarPlayManager.html#/s:16MapboxNavigation14CarPlayManagerC06eventsE0AA0b6EventsE0Cvp">CarPlayManager.eventsManager</a></code> via <code>NavigationEventsManager.init(dataSource:accessToken:mobileEventsManager)</code>, instead of the Info.plist.
+
+</div>
+
+                        <a href="Classes/CarPlayManager.html" class="slightly-smaller">See more</a>
+                      </div>
+                      <div class="declaration">
+                        <h4>Declaration</h4>
+                        <div class="language keyline-blue">
+                          <p  style="background: rgba(65, 131, 196, 0.15);" class="aside-title text-blue">Swift</p>
+                          <pre class="highlight swift"><code><span class="kd">public</span> <span class="kd">class</span> <span class="kt">CarPlayManager</span> <span class="p">:</span> <span class="kt">NSObject</span></code></pre>
+<pre class="highlight swift"><code><span class="kd">extension</span> <span class="kt">CarPlayManager</span><span class="p">:</span> <span class="kt">CPApplicationDelegate</span></code></pre>
+<pre class="highlight swift"><code><span class="kd">extension</span> <span class="kt">CarPlayManager</span><span class="p">:</span> <span class="kt">CPInterfaceControllerDelegate</span></code></pre>
+<pre class="highlight swift"><code><span class="kd">extension</span> <span class="kt">CarPlayManager</span><span class="p">:</span> <span class="kt">CPMapTemplateDelegate</span></code></pre>
+<pre class="highlight swift"><code><span class="kd">extension</span> <span class="kt">CarPlayManager</span><span class="p">:</span> <span class="kt"><a href="Protocols/CarPlayNavigationDelegate.html">CarPlayNavigationDelegate</a></span></code></pre>
+
+                        </div>
+                      </div>
+                      <div class="show-on-github">
+                        <a href="https://github.com/mapbox/mapbox-navigation-ios/tree/v1.2.0-alpha.1/MapboxNavigation/CarPlayManager.swift#L29-L242">
```

Comment:
As @1ec5 explained to me in the previous release, we need to generate the docs from the correct tag, for links to be up-to-date.

Reply:
oh, I see. Then I need to re-visit this once new tag is live.